### DISCUSSION TITLE
[Feature Request] Widget for Issues

### DISCUSSION THREAD

**Discussion by JustACasual** [community/community#146192] [Mobile]
### Select Topic Area

General

### Body

Hi, I think it would be great to have a widget for issues.
Filters could be 
- assigned to me
- created by me
- repo

So I could have different screens on my home to swipe through and see issue lists on which I ought to be working.

When clicking an issue, it could open the mobile app to show the issue, no need for change on that end.

I personally am on Android.

Great work with the mobile app, thank you!

**Comment by HenSquared** [comment]
Thanks for the feedback - will create an issue to track this feature request
